.github/workflows/e2e-release-macos-linux-builds.yaml

```diff
@@ -0,0 +1,27 @@
+name: E2E Test - MacOS and Linux OS Release Builds
```

Comment:
I'm wondering if this Prefix for the action should be changed. This is a test to check the release bits before we publish so it's like an e2e in a way but it's also kinda different and special. I think maybe we should call it something like `Release Gate -` so it's kinda lumped in with the Release actions, but I'm open to ideas and suggestions.

Reply:
I like `Release Gate - `

Reply:
Makes sense, I'll change the GitHub Actions Workflow yaml file name and `name` field and related fields to use `Release Gate` than `E2E Test`. I'm assuming we are okay with bash script file name

Reply:
Updated the PR 👍 Let me know if it makes sense